docs/proofs/output_cpc.rst

```diff
@@ -0,0 +1,35 @@
+Proof format: Cooperating Proof Calculus
```

Comment:
It looks like most of the text here is > 80 char per line.

Reply:
Addressed